temporalio/workflow.py

```diff
@@ -830,6 +881,24 @@ def memo_value(
     return _Runtime.current().workflow_memo_value(key, default, type_hint=type_hint)
 
 
+def get_current_details() -> str:
+    """Get the current details of the workflow which may appear in the UI/CLI.
+    Unlike static details set at start, this value can be updated throughout
+    the life of the workflow and is independent of the static details.
+    This can be in Temporal markdown format and can span multiple lines.
+    """
+    return _Runtime.current().workflow_get_current_details()
+
+
+def set_current_details(description: str) -> None:
```

Comment:
```suggestion
def set_current_details(current_details: str) -> None:
```

Pedantic, but this isn't really a "description" per se